hack/jenkins/job-configs/global.yaml

```diff
@@ -85,6 +85,33 @@
                     <healthScaleFactor>100.0</healthScaleFactor>
                 </hudson.tasks.junit.JUnitResultArchiver>
 
+- builder:
+    name: docker-build-publish
+    builders:
+        - raw:
+            xml: |
+               <com.cloudbees.dockerpublish.DockerBuilder plugin="docker-build-publish@1.1">
+                   <server plugin="docker-commons@1.2"/>
+                   <registry plugin="docker-commons@1.2">
+                       <credentialsId>---</credentialsId>
```

Comment:
I would just hard-code in the value to a variable in the YAML. There are plenty of places where our config can't be ported exactly to other Jenkins instances, so I wouldn't stress this too much.